fcrepo4/fcrepo4
# Triples in mementos not the same as in original resource

There were extra / missing triples in the memento. This fixes that discrepancy between the original and it's newly created memento.
* * *

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2848

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
This makes sure the memento triples are the same as the original triples (for a newly created memento).
# What's new?
* FEDORA_CONTAINER is now added explicitly to the memento.  
* When the memento is requested from the server, it doesn't deliver the server managed triples separately, as they are already part of the PROPERTY triples.  If you get the PROPERTY triples and SERVER_MANAGED triples, you'll get duplication on the server managed properties. 


# How should this be tested?
As per in the JIRA ticket: 

// create a resource to version
$> curl -X POST -H "Link: <http://mementoweb.org/ns#OriginalResource>; rel=\"type\"" http://localhost:8080/fcrepo4/rest/ -H "slug: test1"

// version it
$> curl -X POST http://localhost:8080/fcrepo4/rest/test1/fcr:versions

which creates a memento – fetch memento information by querying the fcr:versions endpoint.

Then get the Original Resource and the memento and compare them: 
$> curl http://localhost:8080/fcrepo4/rest/test1
$> curl http://localhost:8080/fcrepo4/rest/test1/fcr:versions/UNIQUEID

The triples should be the same. 



# Interested parties
@dbernstein @bbpennel
